_partials/_install-self-hosted-docker-based.mdx

```diff
@@ -1,62 +1,214 @@
     ---------------------+---------+------------+---------------------------------------------------------------------------------------
     plpgsql             | 1.0     | pg_catalog | PL/pgSQL procedural language
-    timescaledb         | 2.17.2  | public     | Enables scalable inserts and complex queries for time-series data (Community Edition)
-    timescaledb_toolkit | 1.19.0  | public     | Library of analytical hyperfunctions, time-series pipelining, and other SQL utilities
-    (3 rows)
+    timescaledb         | 2.20.3  | public     | Enables scalable inserts and complex queries for time-series data (Community Edition)
     ```
-    Press q to exit the list of extensions.
+
+    Press `q` to exit the list of extensions.
+
+## More Docker options
+
+If you want to access the container from the host but avoid exposing it to the
+outside world, you can bind to `127.0.0.1` instead of the public interface, using this command:
+
+```bash
+docker run -d --name timescaledb -p 127.0.0.1:5432:5432 \
+-e POSTGRES_PASSWORD=password timescale/timescaledb:latest-pg17
+```
+
+If you don't want to install `psql` and other $PG client tools locally,
+or if you are using a Microsoft Windows host system, you can connect using the
+version of `psql` that is bundled within the container with this command:
+
+```bash
+docker exec -it timescaledb psql -U postgres
+```
+
+Existing containers can be stopped using `docker stop` and started again with
+`docker start` while retaining their volumes and data. When you create a new
+container using the `docker run` command, by default you also create a new data
+volume. When you remove a Docker container with `docker rm`, the data volume
+persists on disk until you explicitly delete it. You can use the `docker volume
+ls` command to list existing docker volumes. If you want to store the data from
+your Docker container in a host directory, or you want to run the Docker image
+on top of an existing data directory, you can specify the directory to mount a
+data volume using the `-v` flag:
+
+```bash
+docker run -d --name timescaledb -p 5432:5432 \
+-v /your/data/dir:/home/postgres/pgdata/data \
+-e POSTGRES_PASSWORD=password timescale/timescaledb:latest-pg17
+```
+
+When you install $TIMESCALE_DB using a Docker container, the $PG settings
+are inherited from the container. In most cases, you do not need to adjust them.
+However, if you need to change a setting, you can add `-c setting=value` to your
+Docker `run` command. For more information, see the
+[Docker documentation][docker-postgres].
+
+The link provided in these instructions is for the latest version of $TIMESCALE_DB
+on $PG 16. To find other Docker tags you can use, see the [Dockerhub repository][dockerhub].
+
+## View logs in Docker
+
+If you have $TIMESCALE_DB installed in a Docker container, you can view your logs
+using Docker, instead of looking in `/var/lib/logs` or `/var/logs`. For more
```

Comment:
I've never heard of `/var/lib/logs`, and it's also not `/var/logs` but `/var/log` :)